intellst-api/src/Services/IdentifiedCaseHandler.php

```diff
@@ -81,4 +102,92 @@ public function updateIdentifiedCaseAllowEntrance(AllowEntranceDTO $dto, Identif
 
         return $errors;
     }
+
+    public function getIdentifiedCase(): array
+    {
+        $user = $this->userHandler->getList();
+        $temperature = $this->returnUserEnterprise($user)->temperature;
+        $data = $this->returnUserEnterprise($user)->restrictionPeriod;
```

Comment:
not need to make 2 requests to `returnUserEnterprise`
```
$dto = $this->returnUserEnterprise($user);
```
then use `$dto->temperature`